documentation/internal/Overview-of-Navigation-&-URL-structure.md

```diff
@@ -85,9 +83,9 @@ Example: [https://balsamiq.cloud/scs7giw/ponj59g/r1D2B](https://balsamiq.cloud/s
 
 ---
 
-## `/applications/[type]/[guid]/[section]/[page]`
+## `/application/[appId]/[sectionName]/p[page]`
```

Comment:
```suggestion
## `/application/[appId]/[sectionName]/[page]`
```

Reply:
Oh, now I see this was intended. Do you think it's better than just the page number? I think I prefer just the number.

Reply:
I thought it was bit nicer since the `[sectionName]` shows a meaningful name, so it would make it quite clear to the user that what section and page we're on: `/applications/1234/product-details/p2`

But I'm not too fussed, it's only a minor aesthetic thing. Shall we see if Andrei has a preference?

Reply:
I think instead of `p1` we should use either just the number of something more declarative like: `Page-1`